### ISSUE TITLE
Mobile support

### ISSUE THREAD

**Issue by manatarms** [joplin/plugin-templates#101]
Thanks for this excellent plugin. 

I was curious if its possible to turn on mobile support for the plugin. It looks like the plugin is "desktop-only" by default. 
https://github.com/laurent22/joplin/issues/10360

Happy to help test and make the code changes required to make that happen.

Cheers!

**Comment by ImperialSquid** [comment]
I'd also love to see this plugin on mobile (it would make bullet journaling much easier if nothing else!)

According to this [forum post]([personalizedrefriger](https://discourse.joplinapp.org/u/personalizedrefriger)) from @personalizedrefrigerator, it looks like templates isn't compatible on mobile since it uses the file system. But as far as I can tell, it only uses the file system to import old templates that used to be stored outside of Joplin. However, all templates are now stored as notes, and it's been that way for (I think) a few years at this point, so I wonder how necessary that code even is anymore...

**Comment by nishantwrp** [comment]
I agree, we can drop legacy templates and logging for a mobile version of this plugin. @laurent22 what do you think of it as a mini-project (could be clubbed with other mini-projects as well) for Google Summer of Code 2025?

**Comment by LightTemplar** [comment]
Still no support? :(

**Comment by shbach** [comment]
Mobile support was added quite a while ago if I'm not mistaken.

**Comment by ImperialSquid** [comment]
Yep, mobile support has been there for a _long_ time

https://github.com/joplin/plugin-templates/pull/110

(This was also around the time the advanced template helpers got added, all part of a big push by the Joplin devs since they took over managing the plugin iirc, it's been a while)

**Comment by LightTemplar** [comment]
Thanks, it took some time before I figured out, where to find inserting of templates in the UI ))
But I found it only in the note toolbar, and I still couldn't find the menu, shown in the #110. Am I miss something?

**Comment by ImperialSquid** [comment]
If you're on mobile: 

If you're viewing the notes in a notebook, the menu is in the puzzle piece shape at the top (this is where all plugin windows go).

If you're editing a note, you can generate from a template using the icon that looks like a sheet of paper (if you don't have it, you may need to enable it, click the gears to change what appear in your tool bar).

If you're on desktop, I don't remember what you need to click sorry, hopefully someone else remembers.

**Comment by LightTemplar** [comment]
I found it all, many thanks! 

Why is this issue still open then, by the way?

**Comment by ImperialSquid** [comment]
Presumably people just forgot to close it when it was implemented 🤷 no specific reason
